### DISCUSSION TITLE
Please add latest tags

### DISCUSSION THREAD

**Discussion by deafmute1** [mealie-recipes/mealie#2669] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Docker convention is the newest stable image is tagged as latest, and this is the tag used when none is specified in compose. I understand that there is only a RC for v1 available right now but this is the version that the documentation currently suggests installing as a "stable" version. This allows me to track the latest stable update when running mass updates across all my docker containers, without manually checking for and bumping version tag.

### Please provide a concise description of the feature that would resolve your issue.

Add `latest` tag to latest built stable docker container in github container registry.

### Please consider and list out some caveats or tradeoffs made in your design decision

N/A.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by JayKuswahey** [comment]
I personally don't care for `latest`, but definitely a +1 for having a `stable` and/or a rolling `v1` (and later `v2` to migrate to 2.x) tag 👍🏼
